src/pages/VideoTemplate/Video.page.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useEffect } from 'react';
+import { useLocation } from 'react-router-dom';
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardMedia from '@material-ui/core/CardMedia';
+import CardContent from '@material-ui/core/CardContent';
+import Typography from '@material-ui/core/Typography';
+import { fetchVideoById } from '../../utils/hooks/useYouTube';
+
+const useStyles = makeStyles({
+  videoDetailsContainer: {
+    maxWidth: '60%',
+    margin: 'auto',
+  },
+  root: {
+    boxShadow: 'none',
+  },
+  media: {
+    height: 400,
+  },
+});
+
+// import { fetchVideoById } from
+const Video = () => {
+  const classes = useStyles();
+
+  const location = useLocation();
+  const currentPathName = location.pathname.split('/')[1];
+  const [videoInfo, setVideoInfo] = useState();
+  useEffect(() => {
+    fetchVideoById(currentPathName).then((response) => {
+      setVideoInfo(response.items[0]);
+    });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
The eslint rule `eslint-disable-next-line react-hooks/exhaustive-deps` exists for one reason, it can alert us for possible bugs or unexpected behaviors on the app.

Please remove this ignore rule comment and try to fix it the eslint message.